packages/amplication-client/src/VersionControl/use-buildWatchStatus.tsx

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useMemo } from "react";
```

Comment:
Filename should be either `useBuildWatchStatus.tsx` or `use-build-watch-status.tsx`